apollo-portal/src/main/resources/static/i18n/zh-CN.json

```
@@ -316,6 +316,8 @@
   "Index.GetCreateAppRoleFailed": "获取创建应用权限信息失败",
   "Index.Topped": "置顶成功",
   "Index.CancelledFavorite": "取消收藏成功",
+  "Index.PublicProject": "公共项目",
+  "Index.PublicTip": "您还没有任何公共项目，在项目配置中可以设置哟~",
```

There is still not a proper translation for `Namespace`, so I suggest we keep the English name here. 
```suggestion
  "Index.PublicProject": "公共Namespace",
  "Index.PublicTip": "您还没有任何公共Namespace，在你的项目中可以创建哟~",
```